roles/oidcng/templates/openid-configuration.json.j2

```diff
@@ -0,0 +1,88 @@
+{
+  "issuer": "https://org.openconext.oidc.ng",
+  "authorization_endpoint": "https://oidcng.{{ base_domain }}/oidc/authorize",
+  "token_endpoint": "https://oidcng.{{ base_domain }}/oidc/token",
+  "userinfo_endpoint": "https://oidcng.{{ base_domain }}/oidc/userinfo",
+  "introspect_endpoint": "https://oidcng.{{ base_domain }}/oidc/introspect",
+  "jwks_uri": "https://oidcng.{{ base_domain }}/oidc/certs",
+  "response_types_supported": [
+    "code",
+    "token",
+    "id_token",
+    "code token",
+    "code id_token",
+    "token id_token",
+    "code token id_token"
+  ],
+  "response_modes_supported": [
+    "fragment",
+    "query",
+    "form_post"
+  ],
+  "grant_types_supported": [
+    "authorization_code",
+    "implicit",
+    "refresh_token",
+    "client_credentials"
+  ],
+  "subject_types_supported": [
+    "public",
+    "pairwise"
+  ],
+  "id_token_signing_alg_values_supported": [
+    "RS256"
+  ],
+  "scopes_supported": [
+    "openid",
+    "groups",
+    "profile",
+    "email",
+    "address",
+    "phone"
+  ],
+  "token_endpoint_auth_methods_supported": [
+    "client_secret_post",
+    "client_secret_basic"
+  ],
+  "claims_supported": [
+    "aud",
+    "nbf",
+    "iss",
+    "exp",
+    "iat",
+    "jti",
+    "nonce",
+    "at_hash",
+    "c_hash",
+    "s_hash",
+    "at_hash",
+    "sub",
+    "edumember_is_member_of",
+    "eduperson_affiliation",
+    "eduperson_entitlement",
+    "eduperson_principal_name",
+    "eduperson_scoped_affiliation",
+    "email",
+    "family_name",
+    "given_name",
+    "name",
+    "nickname",
+    "preferred_username",
+    "schac_home_organization",
+    "schac_home_organization_type",
+    "schac_personal_unique_code",
+    "uids"
+  ],
+  "claims_parameter_supported": true,
+  "acr_values_supported": [
+    "http://{{ env }}.surfconext.nl/assurance/loa1",
+    "http://{{ env }}.surfconext.nl/assurance/loa2",
+    "http://{{ env }}.surfconext.nl/assurance/loa3",
+    "http://{{ env }}.surfconext.nl/assurance/sfo-level2",
+    "http://{{ env }}.surfconext.nl/assurance/sfo-level3"
+  ],
+  "code_challenge_methods_supported": [
+    "plain",
+    "S256"
+  ]
+}
```

Comment:
I reviewed this change in the other PR

Reply:
That's correct. I needed to merge feature branches otherwise the deploy would fail